test-e2e/build/augment-path.test.js

```diff
@@ -0,0 +1,37 @@
+const childProcess = require('child_process');
+const path = require('path');
+const {promisify} = require('util');
```

Comment:
I think this is only available in newer nodes and that means we can use `async`/`await` syntax (we are using it for `test-e2e-esyi` anyways).

Reply:
👍 - this would help remove the `then` chains in the tests